jkamz/ireporter
# 164325106 integrate travis ci

### What does this PR do?
This PR integrates travis CI with the project to aid in running tests automatically

### Description of the tasks to be completed
- Integrate Travis
- Include badge in README

### How can this be manually tested?
Clone this repository on local machine and CD into project folder

`$ git clone https://github.com/jkamz/ireporter`

`$ cd ireporter`

Check out to the PR's branch
`$ git checkout ch-travis-ci-164325106`

### What are the relevant Pivotal Tracker stories?
#### 164325106
#### https://www.pivotaltracker.com/story/show/164325106

### Screeenshots
![screenshot from 2019-03-05 23-36-45](https://user-images.githubusercontent.com/11965186/53835903-9c87b480-3f9f-11e9-9ffd-b3e7e8d7fc48.png)
